.github/workflows/deploy-pypi.yml

```diff
@@ -19,7 +19,7 @@ on:
 
 jobs:
   deploy:
-    if: github.event.pull_request.merged == true || github.event.action == ''
```

Comment:
Was this not working before?

Reply:
It was! I can get rid of the parens, It's a remnant of an earlier commit